src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1754,6 +1754,17 @@ class Simplex_tree {
   /** \brief Upper bound on the dimension of the simplicial complex.*/
   int dimension_;
   bool dimension_to_be_lowered_ = false;
+
+//MULTIPERS STUFF
+public: 
+  void set_number_of_parameters(unsigned int num){
+		number_of_parameters_ = num;
+	}
+	unsigned int get_number_of_parameters() const{
+		return number_of_parameters_;
+	}
+private:
+	unsigned int number_of_parameters_;
```

Comment:
One extra integer is acceptable, but I wonder if we could make it exist or not depending on the template parameter?

Reply:
I didn't find anywhere else to put it. Its not possible on the SimplexTree options (static). I can put it in the python class, but that would imply passing this integer to c++ each time a c++ function is called.

Reply:
One possibility would be `[[no_unique_address]] std::conditional_t<Options::something, int, SomeEmptyType> number_of_parameters_;` so that depending on the option, it would not take any space. Or have Simplex_tree derive from `Maybe_multi_base<SimplexTreeOptions::plouf>` which depending on plouf can be empty or contain an int (and provide 2 methods).
Again, that's not fundamental, I just wanted to mention this possibility.